src/pages/workspace/accounting/qbo/import/QuickbooksTaxesPage.tsx

```diff
@@ -44,10 +46,12 @@ function QuickbooksTaxesPage({policy}: WithPolicyProps) {
                                     accessibilityLabel={translate('workspace.accounting.taxes')}
                                     isOn={!!syncTax}
                                     onToggle={() => Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.SYNC_TAX, !syncTax)}
+                                    disabled={isJournalExportEntity}
```

Comment:
Since it is possible that `syncTax` is true (i.e. along with `Journal Entry` export) due to settings via Expensify Classic or offline scenarios, it may be better to keep it `enabled` when `syncTax` is `true` so that the user can toggle `syncTax` to `false`. However, we can keep it `disabled` when `syncTax` is false and `Journal Entry` is set to prevent the user from toggling. Something like `!syncTax && isJournalExportEntity` could solve this problem. 
What do you think?
Here is a video demonstrating the problem

https://github.com/Expensify/App/assets/3004659/77fbb222-7180-4d98-a092-50077c2661dd

Reply:
I agree with @rojiphil,

although this possibility is very low, it's better to leave the option for the user to correct the misconfiguration than just locking it all the time when the journal entry is selected.

Let's do `!syncTax && isJournalExportEntity` as @rojiphil proposed.